[snap7] official demo build failed

**Describe the bug**
When i tryed to build the snap7's official c++ demo `examples/cpp/client.cpp` with vcpkg's snap7 ports, it failed.

**Environment**
- OS: Windows 10 Pro 22H2 19045.3324
- Compiler: MSVC 19.37.32822.0
- snap7 version: 1.4.2#1

**To Reproduce**
Steps to reproduce the behavior:
1. create a minimium demo project with vcpkg manifest mode, add "snap7" dependency in `vcpkg.json`
2. add the snap7's official c++ demo `examples/cpp/client.cpp` to project
3. add `find_package(snap7 CONFIG REQUIRED)` and `target_link_libraries(main PRIVATE snap7::snap7)` to project's `CMakeLists.txt`
4. build... it failed.

**Expected behavior**
It should be succeed.

**Failure logs**
```
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: __cdecl TS7Client::TS7Client(void)" (??0TS7Client@@QEAA@XZ)，referenced in function main
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: __cdecl TS7Client::~TS7Client(void)" (??1TS7Client@@QEAA@XZ)，referenced in function "public: void * __cdecl TS7Client::`scalar deleting destructor'(unsigned int)" (??_GTS7Client@@QEAAPEAXI@Z)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::ConnectTo(char const *,int,int)" (?ConnectTo@TS7Client@@QEAAHPEBDHH@Z)，referenced in function "bool __cdecl CliConnect(void)" (?CliConnect@@YA_NXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::Disconnect(void)" (?Disconnect@TS7Client@@QEAAHXZ)，referenced in function "void __cdecl CliDisconnect(void)" (?CliDisconnect@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::ReadMultiVars(struct TS7DataItem *,int)" (?ReadMultiVars@TS7Client@@QEAAHPEAUTS7DataItem@@H@Z)，referenced in function "void __cdecl MultiRead(void)" (?MultiRead@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::ListBlocks(struct TS7BlocksList *)" (?ListBlocks@TS7Client@@QEAAHPEAUTS7BlocksList@@@Z)，referenced in function "void __cdecl ListBlocks(void)" (?ListBlocks@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::Upload(int,int,void *,int *)" (?Upload@TS7Client@@QEAAHHHPEAXPEAH@Z)，referenced in function "void __cdecl UploadDB0(void)" (?UploadDB0@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::GetOrderCode(struct TS7OrderCode *)" (?GetOrderCode@TS7Client@@QEAAHPEAUTS7OrderCode@@@Z)，referenced in function "void __cdecl OrderCode(void)" (?OrderCode@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::GetCpuInfo(struct TS7CpuInfo *)" (?GetCpuInfo@TS7Client@@QEAAHPEAUTS7CpuInfo@@@Z)，referenced in function "void __cdecl CpuInfo(void)" (?CpuInfo@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::GetCpInfo(struct TS7CpInfo *)" (?GetCpInfo@TS7Client@@QEAAHPEAUTS7CpInfo@@@Z)，referenced in function "void __cdecl CpInfo(void)" (?CpInfo@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::ReadSZL(int,int,struct TS7SZL *,int *)" (?ReadSZL@TS7Client@@QEAAHHHPEAUTS7SZL@@PEAH@Z)，referenced in function "void __cdecl ReadSzl_0011_0000(void)" (?ReadSzl_0011_0000@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::ExecTime(void)" (?ExecTime@TS7Client@@QEAAHXZ)，referenced in function "bool __cdecl Check(int,char const *)" (?Check@@YA_NHPEBD@Z)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::PDURequested(void)" (?PDURequested@TS7Client@@QEAAHXZ)，referenced in function "bool __cdecl CliConnect(void)" (?CliConnect@@YA_NXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::PDULength(void)" (?PDULength@TS7Client@@QEAAHXZ)，referenced in function "bool __cdecl CliConnect(void)" (?CliConnect@@YA_NXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::PlcStatus(void)" (?PlcStatus@TS7Client@@QEAAHXZ)，referenced in function "void __cdecl UnitStatus(void)" (?UnitStatus@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::SetAsCallback(void (__cdecl*)(void *,int,int),void *)" (?SetAsCallback@TS7Client@@QEAAHP6AXPEAXHH@Z0@Z)，referenced in function main
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: bool __cdecl TS7Client::CheckAsCompletion(int *)" (?CheckAsCompletion@TS7Client@@QEAA_NPEAH@Z)，referenced in function "void __cdecl AsPOUploadDB0(void)" (?AsPOUploadDB0@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::WaitAsCompletion(unsigned int)" (?WaitAsCompletion@TS7Client@@QEAAHI@Z)，referenced in function "void __cdecl AsEWUploadDB0(void)" (?AsEWUploadDB0@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "public: int __cdecl TS7Client::AsUpload(int,int,void *,int *)" (?AsUpload@TS7Client@@QEAAHHHPEAXPEAH@Z)，referenced in function "void __cdecl AsCBUploadDB0(void)" (?AsCBUploadDB0@@YAXXZ)
[build] client.cpp.obj : error LNK2019: unresolved external symbol "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl CliErrorText(int)" (?CliErrorText@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@H@Z)，referenced in function "bool __cdecl Check(int,char const *)" (?Check@@YA_NHPEBD@Z)
[build] src\demo.exe : fatal error LNK1120: 20 unresolved externals
```

**Additional context**
Add any other context about the problem here.

--- a/ports/snap7/CMakeLists.txt
+++ b/ports/snap7/CMakeLists.txt
@@ -37,8 +37,6 @@ add_library(${PROJECT_NAME} SHARED "core/s7_client.cpp"
                                    "sys/sol_threads.h"
                                    "sys/unix_threads.h"
                                    "sys/win_threads.h"
-                                   "../release/Wrappers/c-cpp/snap7.h"
-                                   "../release/Wrappers/c-cpp/snap7.cpp"
                                    )
 
 if(MSVC)
@@ -51,7 +49,6 @@ add_library(${PROJECT_NAME}::${PROJECT_NAME} ALIAS ${PROJECT_NAME})
 target_include_directories(${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/core> $<INSTALL_INTERFACE:include>)
 target_include_directories(${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lib> $<INSTALL_INTERFACE:include>)
 target_include_directories(${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/sys> $<INSTALL_INTERFACE:include>)
-target_include_directories(${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../release/Wrappers/c-cpp> $<INSTALL_INTERFACE:include>)
 
 install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME})
 
@@ -66,5 +63,4 @@ if(NOT DISABLE_INSTALL_HEADERS)
   install(DIRECTORY "core/" DESTINATION include/${PROJECT_NAME} FILES_MATCHING PATTERN "*.h")
   install(DIRECTORY "lib/" DESTINATION include/${PROJECT_NAME} FILES_MATCHING PATTERN "*.h")
   install(DIRECTORY "sys/" DESTINATION include/${PROJECT_NAME} FILES_MATCHING PATTERN "*.h")
-  install(DIRECTORY "../release/Wrappers/c-cpp/" DESTINATION include/${PROJECT_NAME} FILES_MATCHING PATTERN "*.h")
 endif()

--- a/ports/snap7/vcpkg.json
+++ b/ports/snap7/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "snap7",
   "version": "1.4.2",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Snap7",
   "homepage": "https://snap7.sourceforge.net/",
   "license": "LGPL-3.0",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8202,7 +8202,7 @@
     },
     "snap7": {
       "baseline": "1.4.2",
-      "port-version": 1
+      "port-version": 2
     },
     "snappy": {
       "baseline": "1.1.10",

--- a/versions/s-/snap7.json
+++ b/versions/s-/snap7.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "0",
+      "version": "1.4.2",
+      "port-version": 2
+    },
+    {
       "git-tree": "9183c4fc6d65ae695c68f0481e5a7dc0067f65fe",
       "version": "1.4.2",
       "port-version": 1

--- a/versions/s-/snap7.json
+++ b/versions/s-/snap7.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "0",
+      "git-tree": "a5d8bb0bb1a47a7e6c583d30f4c394d23704d130",
       "version": "1.4.2",
       "port-version": 2
     },

